src/eventer/eventer_jobq.c

```diff
@@ -267,6 +267,8 @@ eventer_jobq_create_internal(const char *queue_name, eventer_jobq_memory_safety_
   jobq->mem_safety = mem_safety;
   jobq->isbackq = isbackq;
   jobq->lifo = &jobq_lifo_default;
+  ck_pr_store_8(&jobq->shutdown_state, EVENTER_JOBQ_RUNNING);
```

Comment:
This store doesn't really need to be atomic because no other threads have access to the jobq yet.  But it won't hurt antyhing.

Reply:
I mostly just did this for consistency - make it clear we should always be doing this.